commcare_connect/templates/tailwind/base.html

```diff
@@ -0,0 +1,34 @@
+{% load static i18n %}
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>CommCareConnect</title>
+    {% load static %}
+    {% block css %}
+       <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/flatpickr/dist/flatpickr.min.css">
+       <link rel="stylesheet" href="{% static 'bundles/css/tailwind.css' %}">
+       <link href="https://fonts.googleapis.com/css2?family=Work+Sans:wght@400;500;600;700&display=swap" rel="stylesheet">
+      <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/aquawolf04/font-awesome-pro@5cd1511/css/all.css" />
+       {% endblock %}
+       {% block javascript %}
+       <script src="{% static 'bundles/js/vendors-bundle.js' %}" defer></script>
+       <script src="{% static 'bundles/js/tailwind-bundle.js' %}" defer></script>
+       <script src="https://unpkg.com/htmx.org/dist/ext/loading-states.js"></script>
+       <script src="https://cdn.jsdelivr.net/npm/flatpickr"></script>
+      {% endblock javascript %}
+  </head>
+  <body hx-headers='{"X-CSRFToken": "{{ csrf_token }}"}' class="flex flex-col justify-center pl-16 overflow-x-hidden bg-stone-100 font-work-sans">
+    {% include 'tailwind/layouts/sidenav.html'%}
+    <div class="flex-1  px-4">
+      {% include 'tailwind/layouts/header.html'%}
+      <div class="h-5 w-full mt-16"></div>
+      <main id="content" class="md:max-w-screen-2xl mx-auto">
+        {% block content %}
+        {% endblock %}
```

Comment:
Our current base template includes blocks for messages, modals, and inline js. Do we no longer need those, will they be added elsewhere, or just added later to the base?